pimd/pim_msdp.c

```diff
@@ -1076,12 +1075,12 @@ static enum pim_msdp_err pim_msdp_peer_new(struct pim_instance *pim,
 	mp = XCALLOC(MTYPE_PIM_MSDP_PEER, sizeof(*mp));
 
 	mp->pim = pim;
-	mp->peer = peer_addr;
```

Comment:
don't get the need for re-naming the variables?

Reply:
I changed the parameters type and since I had to change their usage I also made the variable names shorter. I hope this is alright.